hash_practice/exercises.py

```diff
@@ -2,18 +2,42 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n log n)
+        Space Complexity: O(n)
     """

```

Comment:
👍   This would be O(n * m log m) for time complexity if the words could be any length.  This is because you are traversing `n` words and sorting each one.